process/working_groups.md

```diff
@@ -0,0 +1,21 @@
+# ASWF Working Groups
+_Modeled off of CNCF Working Group model ( https://github.com/cncf/toc/blob/master/workinggroups/README.md )_
+
+## Introduction
+
+The purpose of working groups are to study and report on a particular question and make recommendations based on its findings. The end result of a working group may be a new project proposal, landscape, whitepaper or even a report detailing their findings. The intention of working groups is not to host a full project or specification. Working Groups can be formed at any time but must be sponsored by a [TAC member] and voted with a super majority vote by the ASWF TAC. The TAC can also shut down a working group with a super majority vote.
```

Comment:
its findings -> their findings (since sentence started out with plural "working groups"). 
second sentence should end in its findings since this now has a single working group

Have we defined anywhere else what a "super majority vote" means in the context of the TAC?

Reply:
Good point raised @jfpanisset - looking at the lifecycle the TAC only requires affirmative votes to progress through the lifecycle stages, so I would think the same should apply here.